containers/test-apps/kitchen/bin/kitchen

```diff
@@ -662,34 +690,9 @@ class Kitchen(HTTPWebSocketsHandler):
 
         # Check if client requested echo server mode
         echo_server = self.headers.get('x-kitchen-echo') is not None
-        echo_buffer = bytearray() if echo_server else None
```

Comment:
Replacing this line with `self.__request_buffer = bytearray()` changes the behavior of the program. The `__slurp_bytes` helper could previously process a request with size _O(n)_ using _O(1)_ memory, but now it needs _O(n)_ memory because you always initialize this to a `bytearray`.

Reply:
Thanks. Changed the code to avoid storing the entire request body in memory.